rpgpy/spcutil.py

```diff
@@ -129,3 +129,46 @@ def find_peak_edges(signal: np.array) -> Tuple[int, int]:
 
     return edge_left, edge_right
 
+
+def spectral_LDR_Galetti(spec_HV: np.array, spec_H: np.array, spec_VH_Re:np.array, spec_VH_Im: np.array,
+                         noise_HV: np.array, noise_H: np.array, num_FFT: np.array, num_rangebins: int,
+                         range_offsets: np.array, version: int) -> np.array:
+    """
+    Compute spectral (S)LDR for vertically pointing radar according to the method by Galetti et al. (2012);
+    Based on code by Alexander Myagkov (RPG).
+    For RPG radars software version > 5.40, the combined spectrum is normalized by 4 (previously 2)
+    Args:
+        spec_HV: H+V channel reflectivity spectra
+        spec_H: only horizontal channel power spectra
+        spec_VH_Re: covariance spectrum Re
+        spec_VH_Im: covariance spectrum imaginary part
+        noise_V: noise power V channel
+        noise_HV: noise power H+V channel
+        num_FFT: number of FFT points
+        num_rangebins: number of range layers
+        range_offsets: range offsets
+        version: software version of RPG radar
+
+    Returns:
+        np.array: SLDR array in dB units
+
+    """
+    if version < 5.40:
+        spec_V = 2 * spec_HV - spec_H - 2 * spec_VH_Re
+    else:
+        spec_V = 4 * spec_HV - spec_H - 2 * spec_VH_Re
```

Comment:
Is this the correct place for the scaling (2 * or 4 *)? It should be here if this is the **only** place where we need it. If other methods need this do, it should be done already in `read_rpg()`. In any case, you should write this `scale = 2 if version < 5.40 else 4` and then `spec_V = scale * spec_HV - spec_H - 2 * spec_VH_Re` to avoid repetition.

Reply:
Good suggestion! I put it in an extra function because I'm quite sure we'll need this later on.